package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Selenium project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "protractor protractor_testing/conf/conf.js",
+    "start": "node selenium_testing/test/bbcNewsSearch.js",
```

Comment:
Please rename this script: you should also **test** something with Selenium Webdriver.

Reply:
Done. Added with &&